pdd.py

```diff
@@ -36,7 +36,7 @@ class _CursorInfo(ctypes.Structure):
 
 monthdict = {name.lower(): num
              for num, name in enumerate(cal.month_abbr) if num}
-_VERSION_ = '1.3'  # current program version
+_VERSION_ = '1.3.1'  # current program version
```

Comment:
Would this be updated in pypi without adding a tag? Otherwise, we will have to wait for the next release.

Reply:
I would suggest you keep it 1,3 as of now and we change it when we make the next release.

Reply:
It's not possible to update the package in PyPI without changing the version.

I would favor releasing a [new patch version](https://semver.org/) as this would allow me to package *pdd* for *openSUSE*.

Reply:
Please leave it as 1.3 and I will try to make a new patch release as 1.3.1 in a few days. I need to see if I can add any feature too. Please suggest if you have any feature in mind.

Reply:
Alright, no worries. I don't have ideas for features. I find pdd fine as it is.